recipes/kafka.rb

```diff
@@ -34,7 +34,7 @@
   elsif node['cdap']['cdap_site'].key?('kafka.log.dir')
     node['cdap']['cdap_site']['kafka.log.dir']
   else
-    '/tmp/kafka-logs'
+    '/data/cdap/kafka-logs'
   end
 
 node.default['cdap']['cdap_site']['kafka.server.log.dirs'] = kafka_log_dirs
```

Comment:
That's a big change from every other place we've used code like this. If we change this, we should change it everywhere.